src/rust/src/lazyframe/general.rs

```diff
@@ -53,7 +52,7 @@ impl PlRLazyFrame {
         cluster_with_columns: bool,
         streaming: bool,
         _eager: bool,
-    ) -> Result<Self> {
+    ) -> Result<PlRLazyFrame> {
```

Comment:
Why?

Reply:
@etiennebacher Your change covers all functions, so don't just fix one and mark it as resolved.
Why did you change everything?

Reply:
Got confused as in the codebase sometimes we specify the output type instead of `Self`, reverted